src/task_8_3.py

```diff
@@ -0,0 +1,26 @@
+# Описать функцию Sin1( x , ε ) вещественного типа (параметры x , ε — вещественные, ε > 0),
+# находящую приближенное значение функции sin( x ):
+# sin( x ) = x – x ^3 /(3!) + x^ 5 /(5!) – ... + (–1) ^ n · x^( 2·n+1) /((2· n +1)!) + ... .
+# В сумме учитывать все слагаемые, модуль которых больше ε .
+# С помощью Sin1 найти приближенное значение синуса для данного x при шести данных ε .i
+
+import math
+
+
+def Sin_1(x, eps):
```

Comment:
```suggestion
def sin_1(x, eps):
```
Please add typing and docstrings